src/components/About/About.js

```diff
@@ -1,10 +1,95 @@
 import React from 'react';
+import { 
+  Octokit, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5619303)